public/assets/js/agenda-plus.js

```diff
@@ -0,0 +1,5 @@
+function toggle() {
+    let visibleArray = (document).getElementsByClassName("visible");
+    for (let i = 0; i < visibleArray.length; i++)
+        visibleArray.item(i).classList.toggle('d-none');
+}
```

Comment:
So now we have a function never used ... :-(. Is there a reason to keep it ?

Reply:
ok.. i have delete all elements about js script for agenda